src/baskerville/scripts/hound_snp_gcs.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+# Copyright 2023 Calico LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========================================================================
+from optparse import OptionParser
+import pickle
+import os
+import tempfile
+import shutil
+import tensorflow as tf
+from baskerville.helpers.gcs_utils import (
+    upload_folder_gcs,
+    is_gcs_path,
+    download_rename_inputs,
+)
+from baskerville.snps import calculate_sad
```

Comment:
@lruizcalico this function is not found in `baskerville.snps`, can you add/update